frontend/src/pages/myPostsPage.jsx

```diff
@@ -110,8 +123,24 @@ class myPostsPage extends React.Component {
                     description={<div style={{textAlign:'left'}}>{item.author + ':  '}{item.description}</div>}
                   />
                   <div style={{textAlign: 'left', marginTop: '10px'}}>
-                  {item.content}
+                  {item.contentType==="image/png;base64" || item.contentType==="image/jpeg;base64"?
+                  <img src={item.content} height="80%" width="80%"></img>:item.content}
```

Comment:
should the case after the `:` just be null? since we handle otherr types differently